Quadriga/src/main/webapp/WEB-INF/views/public/searchednetwork.jsp

```diff
@@ -0,0 +1,215 @@
+<%@ page language="java" contentType="text/html;"%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+
+
+<script
+	src="${pageContext.servletContext.contextPath}/resources/js/jquery-1.9.1.min.js"></script>
+<script
+	src="${pageContext.servletContext.contextPath}/resources/js/cytoscape/dist/cytoscape.js"></script>
+
+
+<div class="row">
+	<div class="alert alert-info" id="load">Searching for network
+		containing the term ${searchNodeLabel}...</div>
```

Comment:
Should be simply "Searching for ${searchNodeLabel}. This might take a little while, please wait..."